pages/stats.tsx

```diff
@@ -0,0 +1,31 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+
+import { LeftSidebarLayout } from '@/src/components/layout/LeftSidebarLayout'
+import { SectionIntro } from '@/src/components/section/SectionIntro'
+import List from '@/src/components/stats/List'
+
+const Stats: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>Aelin - Stats</title>
+      </Head>
+      <LeftSidebarLayout>
+        <SectionIntro
+          backgroundImage="/resources/svg/bg-sponsors.svg"
+          backgroundPosition="100% 100px"
+          title="Stats"
+        >
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+          ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+          ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
```

Comment:
Looks like a placeholder text, do we have the wording?

Reply:
yeah good catch. forgot about that one lol